tracer/src/Datadog.Trace/Propagators/B3MultipleHeaderContextPropagator.cs

```diff
@@ -73,9 +73,11 @@ private B3MultipleHeaderContextPropagator()
 
         internal static void CreateHeaders(SpanContext context, out string traceId, out string spanId, out string sampled)
         {
-            var samplingPriority = context.TraceContext?.SamplingPriority ?? context.SamplingPriority;
-            sampled = samplingPriority > 0 ? "1" : "0";
+            // BUG: we should fall back to SamplingPriorityValues.AutoKeep
```

Comment:
Yes.

Reply:
Sorry, someone was looking over my shoulder and thought this would be funny. Seriously, though...

----

> Or that the fallback should be _instead_ of `context.SamplingPriority`?

Ideally, yes. This fallback is only there for tests that create stand-alone `SpanContext`. Here's one example: https://github.com/DataDog/dd-trace-dotnet/blob/6566af857fb457b4d13fb6a91ef634f1cf5a887b/tracer/test/Datadog.Trace.Tests/Propagators/B3SingleHeaderPropagatorTests.cs#L33-L39

This is a side-effect of us overloading `SpanContext` and using it both for remote contexts extracted from headers and for local spans. (If you've heard this before, it's part of my `SpanContext` rant.)

----

>  Any reason we shouldn't just do that now? Do tests break? 

Yes. Here's one example of a test (just a few lines below the one linked above) that would break because it expects the trace to be not sampled (the `-0` at the end of the header value), when in fact that trace should be kept by using the `AutoKeep` default. This never really happens in real code, so the test is not testing anything useful.

https://github.com/DataDog/dd-trace-dotnet/blob/6566af857fb457b4d13fb6a91ef634f1cf5a887b/tracer/test/Datadog.Trace.Tests/Propagators/B3SingleHeaderPropagatorTests.cs#L42-L46

----

> do you have a separate PR for it?

Yes. I intentionally wanted to change only formatting and such in this PR without any visible behavior changes.